bcbio/variation/vcfutils.py

```diff
@@ -324,6 +330,25 @@ def _do_merge(orig_files, out_file, config, region):
         bgzip_and_index(out_file, config)
     return out_file
 
+def _do_combine_variants(orig_files, out_vcf, ref_file, config, region, minimumN = 3):
```

Comment:
Is the GATK3 dependency really necessary? This makes matters complicated due to it only supporting ancient Java.

Reply:
Agree, I was going to get rid of gatk3 dependency as well. Unfortunately, it turned out not to be very straightforward (see purecn github - there was a discussion on that). what gatk3 is doing here is not supported directly by bcftools or gatk4/picard. It is just (1) merging vcfs into a multisample vcf, (2) filtering sites having at least 3 calls (in any of the samples). (2) is an issue. For example, it is supported in bcftools with plugins, but plugins don't get installed via conda: https://github.com/bioconda/bioconda-recipes/issues/23993
gatk3 goes open-source https://github.com/bioconda/bioconda-recipes/pull/7138, so licensing is not an issue anymore.